exercises/list-ops/example.rb

```diff
@@ -0,0 +1,67 @@
+class ListOps
```

Comment:
The markdown document said to not use existing functions. I just wanted to make sure it was very obvious that you were writing your own methods and not just using array enumerators to do it.

Here is the entire .md document:
"In functional languages list operations like length, map, and reduce are very common. Implement a series of basic list operations, without using existing functions."